code/modules/mob/living/silicon/robot/robot_modules/station.dm

```diff
@@ -789,6 +741,10 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/shockpaddles/robot/jumper(src)
 	src.modules += new /obj/item/weapon/melee/baton/slime/robot(src)
 	src.modules += new /obj/item/weapon/gun/energy/taser/xeno/robot(src)
+	src.modules += new /obj/item/device/xenoarch_multi_tool(src)
+	src.modules += new /obj/item/weapon/pickaxe/excavationdrill(src)
+	src.modules += new /obj/item/device/cataloguer(src)
+
```

Comment:
What about a depth analysis scanner for the xenoarch portion?

Reply:
Xenoarch multitool contains a depth analysis scanner (as well as alden-saraspova counter and measuring tape)

Reply:
Oh, nice. Didn't realise that.